ChangeLog

```diff
@@ -28,6 +28,9 @@ Bugfix
      a file in pk_sign program. Found by kevlut in #1142.
    * Fix invalid buffer sizes passed to zlib during record compression and
      decompression.
+   * Added the macro MBEDTLS_X509_MAX_FILE_PATH_LEN that enables the user to
```

Comment:
Consider rebasing on the latest release, to lessen the risk we add this message to the previous release's section in the Changelog.